src/pages/Chat/ResetButton.tsx

```diff
@@ -1,15 +1,31 @@
-import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
 import * as React from 'react'
+import styled from 'react-emotion'
 import { ChatContainer } from './state'
 
+const Wrapper = styled('div')({
+  paddingRight: 24,
```

Comment:
I would like this to be marginRight: 26 to be consistent with the other buttons:)

Reply:
and a width of 20%, breaking at 850px to 33%, and at 600px to 50%

Reply:
first comment = 👍
2nd i don't understand. this is what it looks like;
![screenshot 2018-10-25 at 13 52 52](https://user-images.githubusercontent.com/3954425/47498359-51290100-d85d-11e8-983e-f9274020fd86.png)

Reply:
Ah, sry not so clear comment. Those are the breakpoints for when the progress steps gets replaced by a single step, and lastly not showing progress step at all for smallest size

Reply:
but why do we need any width in the first place?